templates/cluster-template-development.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  name: dev-test
```

```suggestion
  name: ${CLUSTER_NAME}
```
Note: We keep the class hard coded, but not the cluster name. For tilt it doesn't matter but in other use-cases it will be more useful if the cluster name can be changed.